docroot/themes/custom/uids_base/uids_base.theme

```diff
@@ -1168,6 +1172,30 @@ function uids_base_preprocess_superfish_menu_items(array &$variables) {
   }
 }
 
+/**
+ * Implements hook_theme().
+ */
+function uids_base_theme($existing, $type, $theme, $path) {
+  return [
+    'field__field_person_email' => [
+      'template' => 'field--node--field-person-email',
+      'base hook' => 'field',
+    ],
+    'field__field_person_education' => [
+      'template' => 'field--node--field-person-education',
+      'base hook' => 'field',
+    ],
+    'field__field_person_phone' => [
+      'template' => 'field--node--field-person-phone',
+      'base hook' => 'field',
+    ],
+    'field__node__field_person_website__person' => [
+      'template' => 'field--node--field-person-website--person',
+      'base hook' => 'field',
+    ],
+  ];
+}
+
```

Comment:
```suggestion
```
I don't think we need this here. I believe these will get picked up automatically in the theme.